### PR TITLE
Solution added

### PR DESCRIPTION
**Pull request by gkhrshch**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .idea/*
 *.iml
 .target/*
-.webapp/*
+*.log
```
Comment (Sofasmile) [inline .gitignore:4]: add a blank line at the end of the file

File: pom.xml [hunk 2]
```diff
@@ -25,13 +25,31 @@
...
+            <groupId>jstl</groupId>
+            <artifactId>jstl</artifactId>
+            <version>1.2</version>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
+
```
Comment (Sofasmile) [inline pom.xml:44]: remove ine line

File: src/main/java/mate/academy/internetshop/Main.java [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 import mate.academy.internetshop.model.Item;
 import mate.academy.internetshop.model.User;
 import mate.academy.internetshop.service.BucketService;
 import mate.academy.internetshop.service.ItemService;
 import mate.academy.internetshop.service.OrderService;
 import mate.academy.internetshop.service.UserService;
+import org.apache.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = Logger.getLogger(InjectorOld.class);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/Main.java:14]: do we need main class? remove it from repository

File: src/main/java/mate/academy/internetshop/dao/impl/UserDaoImpl.java [hunk 4]
```diff
@@ -27,7 +27,7 @@ public User get(Long id) {
     @Override
     public User update(User toUpdate) {
         User user = get(toUpdate.getId());
-        user.setBucket(toUpdate.getBucket());
+        //user.setBucket(toUpdate.getBucket());
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/impl/UserDaoImpl.java:30]: do we need this line?

File: src/main/java/mate/academy/internetshop/model/Item.java [hunk 5]
```diff
@@ -41,4 +41,9 @@ public String toString() {
                 + name + "', price="
                 + price + "}";
     }
+
+    @Override
+    public int compareTo(Item item) {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/model/Item.java:46]: do we need this method?

File: src/main/java/mate/academy/internetshop/service/impl/ItemServiceImpl.java [hunk 6]
```diff
@@ -22,6 +26,15 @@ public Item get(Long id) {
         return itemDao.get(id);
     }
 
+    @Override
+    public List<Item> getAllItems() {
+        List<Item> allItems = Storage.items.stream()
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/service/impl/ItemServiceImpl.java:31]: we can't use Storage in service layer

File: src/main/java/mate/academy/internetshop/service/impl/UserServiceImpl.java [hunk 7]
```diff
@@ -38,4 +39,9 @@ public User delete(Long id) {
     public List<Order> getOrders(Long userId) {
         return userDao.get(userId).getOrders();
     }
+
+    @Override
+    public List<User> getAllUsers() {
+        return Storage.users;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/service/impl/UserServiceImpl.java:45]: :)

File: src/main/webapp/WEB-INF/web.xml [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>registration</servlet-name>
+        <url-pattern>/servlet/registration</url-pattern>
+    </servlet-mapping>
+
+    <listener>
+        <listener-class>mate.academy.internetshop.controller.InjectInitializer</listener-class>
+    </listener>
+</web-app>
```
Comment (Sofasmile) [inline src/main/webapp/WEB-INF/web.xml:109]: add a blank line

File: src/main/java/mate/academy/internetshop/model/Item.java (hunk 5 above)
Comment (gkhrshch) [reply src/main/java/mate/academy/internetshop/model/Item.java:46]: Item model class should implement Comparable interface and have compareTo() method overriden so I can sort Item list by alphabetical order in getAllItems() method

File: src/main/java/mate/academy/internetshop/dao/impl/UserDaoImpl.java (hunk 4 above)
Comment (gkhrshch) [reply src/main/java/mate/academy/internetshop/dao/impl/UserDaoImpl.java:30]: I am going to uncomment it after bucket bound to user is available in webapp version

**Pull request merged**